src/components/composer/composer/composer.ts

```diff
@@ -647,14 +660,20 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
     this.autoCompleteState.showProvider = false;
     this.functionDescriptionState.showDescription = false;
 
+    const dataValidationAutocompleteValues =
+      this.env.model.getters.getAutoCompleteDataValidationValues();
+    if (!content.startsWith("=") && dataValidationAutocompleteValues.length) {
```

Comment:
what if the Autocomplete values all start with = ?
Shouldn't it just be "if datavalidation stuff then show it"?